CHANGES.txt

```diff
@@ -24,6 +24,12 @@ Changes
 Fixes
 =====
 
+- Fixed an issue where attempting to collect documents (using the common
```

Comment:
I think this contains too much implementation detail for end-users. Would keep it on a higher level (e.g. just `Fixed a race condition that caused DELETE or UPDATE statements to result in an error if a shard was being relocated or in a RECOVERY state`)

(SELECT shouldn't have been affected, because the wormhole mechanism isn't used in that case?)